superset/assets/javascripts/explore/stores/getLanguage.js

```diff
@@ -0,0 +1,24 @@
+
+const $ = window.$ = require('jquery');
+
+export function getLanguage() {
+  function getLocale() {
+    const locale = $.ajax({
+      url: '/superset/rest/api/getLocale',
+      async: false,
+    });
+    return JSON.parse(locale.responseText).language;
+  }
+  switch (getLocale()) {
```

Comment:
Why is the directory of the Spanish translation file not in sp/LC_MESSAGES /, and in es/LC_MESSAGES /, and if so, the "en" returned by json.parse (locale. ResponseText), no en/LC_MESSAGES/directory, is not good for translating English.